server_config.toml

```diff
@@ -150,7 +150,7 @@ listener = "tcp://0.0.0.0:46658"
 # This file needs to be in the root directory
 private_validator_file = "priv_validator.json"
 
-  # Tendermint requires additional configuration parameters.
+  # Tendermint requires additional configuration parameters.TransactAndHold
```

Comment:
yeah, while I was typing that, it makes sense to me as well that having cm define the config and edb consume it is dangerous.  Better for cm to have it write and manage all, but depend on a eris-client functionality